app/src/main/java/edu/uc/it3048/mypokedex/ProfileScreenActivity.kt

```diff
@@ -41,15 +41,16 @@ class ProfileScreenActivity : AppCompatActivity() {
     private val CAMERA_REQUEST_CODE = 918
     private val GALLERY_REQUEST_CODE = 555
     private lateinit var loginProviders : List<AuthUI.IdpConfig>
-    private var firestore : FirebaseFirestore = FirebaseFirestore.getInstance()
-    private var storageReference = FirebaseStorage.getInstance()
```

Comment:
Removed unused lines of code. These might get used later though, so if that is the case, ignore this change.